app/controllers/rating_questions_controller.rb

```diff
@@ -1,5 +1,70 @@
+# require 'pry'
+
 class RatingQuestionsController < ApplicationController
+  before_action :set_rating_question, only: [:show, :update, :destroy]
+
   def index
     @rating_questions = RatingQuestion.all
   end
+
+  def new
+    @rating_question = RatingQuestion.new
+  end
+  
+  def question_params
+    params.require(:rating_question).permit(:title, :tag)
+  end
+
+  def create
+   if request.body.size.zero?
+    return render json: {}, status: 400
+   end
+    @rating_question = RatingQuestion.create(question_params)
+    if @rating_question.save
+      respond_to do |format|
+        format.html { redirect_to "/rating_questions", notice: "Your question has been created." }
```

Comment:
It might be better to `redirect_to "/"` here, given that the `root` route for your application is defined as going to `rating_questions#index`, which is the same controller action as `/rating_questions`.

Reply:
thats ok for me to change.